index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&amp;display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css"
+    />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+        <header>
+            <a href="/" class="logo">My Logo</a>
+            <div class="menu" id="icon"><i class="fa fa-bars "></i></div>
+        </header>
+        <main>
+          <section class="section-1" id="Portfolio">
+              <h1>  I’m Lalla <br>
+                  Glad to see you!</h1>
+           <p class="Descriptions"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me. </p>     
+            <a href="#" class="connect">LET'S CONNECT</a>
+            <br><br>
+            <img src="image/social.png">
```

Comment:
- [ ] Please consider adding social media icons as a list of images instead of one image. It should be something like this 
```
<ul>
  <li>
    <a href="my_linkdin_link">
      <img src="" alt="Linkdin icons" />
    </a>
  </li>
  <li>
    <a href="/my_twitter_link">
      <img src="" alt="Twitter icons" />
    </a>
  </li>
  ...
</ul>
```

This will help you improve the semantics of the project. It also allows you to connect each image with your social media link 📱 😺